User incremental search

y-kazuya opened the pull request:
Chatspace グループ作成時のユーザーのインクリメンタルリサーチの実装

Commit:
Merge branch 'master' into user_incremental_search

Commit:
Chatspace グループ作成時のユーザーのインクリメンタルリサーチの実装

Commit:
インデント修正

ShuheiTsuji commented on app/views/groups/_form.html.haml:
```diff
@@ -11,32 +11,22 @@
-      <div id='user-search-result'></div>
-      </div>
+    .chat-group-form__field--left
+      %label.chat-group-form__label{for: "user-search-field"} チャットメンバーを追加
```
こちら form_for のヘルパーメソッドを使用してコードを書きましょう！

ShuheiTsuji commented on app/views/groups/_form.html.haml:
```diff
@@ -11,32 +11,22 @@
-        </div>
+      #chat-group-users
+        #chat-group-user-22.chat-group-user.clearfix
+          %input{name: "chat_group[user_ids][]", type: "hidden", value: "22"}/
```
こういった箇所もform_forで書く方が良いです！

ShuheiTsuji commented on app/views/users/index.json.jbuilder:
```diff
@@ -0,0 +1,4 @@
+json.array! @users do |user|
+  json.id user.id
+  json.name user.name
+end
```
最後に改行をしましょう！

Commit:
[修正]chatspace グループ作成時のユーザのインクリメンタル検索

jbuilderファイルの最後の所に改行を加えた
グループ作成フォームでのチャットメンバーのテキストフォームとそのラベルをform_forのヘルパーメソッドで書いた
あと一点ご指摘いただいている。
25行目から31行目をform_forを使って書き換えろはそもそもラベルつける意味とか、わからなかったし、うまくできなかった

ShuheiTsuji commented on app/assets/javascripts/search.js:
```diff
@@ -0,0 +1,67 @@
+			alert("ユーザー検索に失敗しました");
+		});
+	});
+});
```
ここも改行が必要ですね。

ShuheiTsuji commented on app/views/groups/_form.html.haml:
```diff
@@ -11,32 +11,23 @@
   .chat-group-form__field.clearfix
     .chat-group-form__field--left
-      %label.chat-group-form__label{:for => "chat_group_チャットメンバー"} チャットメンバー
+      %label.chat-group-form__label{for: "chat_group_チャットメンバー"} チャットメンバー
```
ここもform_forのヘルパーメソッドを使用して書くことができますね。

Commit:
[修正2]

search.js 最後に改行
_form.html.haml
   from_forのヘルパーメソッドを用いて書き換えた
